installer/bundle/bundle_skel.sh

```diff
@@ -801,6 +802,16 @@ case "$installMode" in
                     [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
                 fi
             done
+			echo "----- Installing MySQL ruby gem -----"
+			set +e
+			$OMSAGENT_RUBY_DIR/gem install mysql2 2>/dev/null
```

Comment:
Yes, we always want to install, the gem install validates the pre-reqs automatically so there is no need to duplicate that logic in the script. Regardless pre-reqs are met or not, the install takes less than a second . Also the code looks simplier.